ProblemsSolution/ProblemsSolution.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System;
+
+namespace ProblemsSolution
+{
+    public class ArrayProblemsSolution
+    {
+        public static int SumArrayEvenNumbers(int[] numbers)
+        {
+            int sum = 0;
+            foreach(int number in numbers)
+            {
+                if(number % 2 == 0) sum += number;
+            }
+
+            return sum;
+        }
+
+        public static int[] BigestTwoNumbersArray(int[] numbers)
+        {
+            if(numbers.Length == 0) return new int[] {};

```

Comment:
Why do you need this guard in place?

Reply:
> Why do you need this guard in place?

Because otherwise the method would return an array that contains 2 elements with value of Int32.MinValue.